src/px-script/main/alert/index.js

```diff
@@ -7,39 +7,37 @@ const _hideAlert = alertElement => {
     alertElement.classList.add("d-none");
 };
 
-const alert = (() => {
-    const init = () => {
-        const closeButtons = document.querySelectorAll(SELECTORS.CLOSE);
-
-        if (closeButtons) {
-            closeButtons.forEach(btn => {
-                const dataVal = btn.dataset.alertClose;
-
-                if (dataVal) {
-                    const alert = document.getElementById(dataVal);
-
-                    if (alert) {
-                        btn.addEventListener("click", e => {
-                            e.preventDefault();
-                            _hideAlert(alert);
-                        });
-                    } else {
-                        console.warn(`No alert with with id ${dataVal} was found, make sure the attribute data-alert-close contains the correct id.`);
-                    }
+const init = () => {
+    const closeButtons = document.querySelectorAll(SELECTORS.CLOSE);
 
-                } else {
-                    const alert = btn.closest(".alert");
+    if (closeButtons.length) {
+        closeButtons.forEach(btn => {
+            const dataVal = btn.dataset.alertClose;
+
+            if (dataVal) {
+                const alert = document.getElementById(dataVal);
 
+                if (alert) {
                     btn.addEventListener("click", e => {
                         e.preventDefault();
                         _hideAlert(alert);
                     });
+                } else {
+                    console.warn(`No alert with with id ${dataVal} was found, make sure the attribute data-alert-close contains the correct id.`);
                 }
-            });
-        }
-    };
 
-    return { init };
-})();
+            } else {
+                const alert = btn.closest(".alert");
 
-export default alert;
+                btn.addEventListener("click", e => {
+                    e.preventDefault();
+                    _hideAlert(alert);
+                });
+            }
+        });
+    }
+};
+
+export default {
+    init
```

Comment:
const SELECTORS.ALERT are not used by this module, only SELECTORS.CLOSE

Reply:
Removed.